spec/koans.js

```diff
@@ -8,31 +8,32 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+        varX= true;
```

Comment:
the point for this test is to check the differences between the let and the var keywords.
If you do :
```js
    if (true) {
      let varX = true;
      }
```
varX will not evaluate to rue, because let is block scoped, so it exists only inside the if statement.
Whereas if you write :
```js
if (true) {
       var varX = true;
      }
```
it will work, because var is function scoped, that means that it ignores if statements and for loops, the only rule is that is can be accessed anywhere inside the function where it was declared, or anywhere in the file if it was not declared inside a function.